src/Tizen.Applications.Common/Tizen.Applications/ApplicationManager.cs

```diff
@@ -170,6 +171,31 @@ public static async Task<IEnumerable<ApplicationInfo>> GetInstalledApplicationsA
             });
         }
 
+        /// <summary>
+        /// Terminates the application if it is running on background.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of system error.</exception>
+        /// <privilege>http://tizen.org/privilege/appmanager.kill.bgapp</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public static void TerminateBackgroundApplication(ApplicationRunningContext app)
```

Comment:
Let's add remark.

> This function returns after it just sends a request for terminating a background application.
Platform will decide if the target application could be terminated or not according to the state of the target application.